index.html

```diff
@@ -44,10 +44,10 @@ <h1>article header h1</h1>
                     </header>
                     <section>
                         <h2>article section h2</h2>
-                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam sodales urna non odio egestas tempor. Nunc vel vehicula ante. Etiam bibendum iaculis libero, eget molestie nisl pharetra in. In semper consequat est, eu porta velit mollis nec. Curabitur posuere enim eget turpis feugiat tempor. Etiam ullamcorper lorem dapibus velit suscipit ultrices. Proin in est sed erat facilisis pharetra.</p>
+                        <p>El viernes se rejistro una llubia de hamsters</p>
```

Comment:
Please remove typo error in "rejistro" and "llubia", write "registro" instead of "rejistro" and "lluvia" instead of "llubia"

Reply:
done